doc/classes/EditorSettings.xml

```diff
@@ -611,6 +615,9 @@
 		<member name="interface/editor/save_each_scene_on_quit" type="bool" setter="" getter="">
 			If [code]false[/code], the editor will save all scenes when confirming the [b]Save[/b] action when quitting the editor or quitting to the project list. If [code]true[/code], the editor will ask to save each scene individually.
 		</member>
+		<member name="interface/editor/save_on_focus_loss" type="bool" setter="" getter="">
+			If [code]true[/code], scenes and scripts are saved when the editor loses focus. This can be used for a less intrusive autosave behavior than automatically saving on regular intervals.
```

Comment:
> Granted, it is only for text (for some reason) but it is what the sentence is likely referring to.

Yes, this was indeed the setting I was referring to (which lacks a counterpart for scenes… likely because scenes can take a while to save).